platform/frontend/src/app/settings/account/page.tsx

```diff
@@ -1,13 +1,18 @@
 "use client";
 
-import { SecuritySettingsCards } from "@daveyplate/better-auth-ui";
+import { ApiKeysCard, SecuritySettingsCards } from "@daveyplate/better-auth-ui";
 import { Suspense } from "react";
 import { ErrorBoundary } from "@/app/_parts/error-boundary";
 import { LoadingSpinner } from "@/components/loading";
 
 function AccountSettingsContent() {
   return (
     <div className="mx-auto max-w-7xl px-4 py-6 md:px-8 space-y-6 w-full">
+      <ApiKeysCard
+        classNames={{
+          base: "w-full",
+        }}
+      />
```

Comment:
this was accidentally removed [here](https://github.com/archestra-ai/archestra/commit/ceb602bc57332a995e24dce3c81e8b6d5d1f5492#diff-d991989b2e45d29fb1ea524c28fec801f36f6970d8eb80d2f1e8ebeed55e7213L23-L27)